scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -888,7 +888,7 @@ cleanup_joiner()
         wsrep_log_debug "Cleaning up fifo file $progress"
         rm $progress
     fi
-    if [[ -n "${JOINER_SST_DIR}" && -z "$WSREP_LOG_DEBUG" ]]; then
+    if [[ -n "${JOINER_SST_DIR-}" && -z "$WSREP_LOG_DEBUG" ]]; then
```

Comment:
Not a strong opinion. Can we use `:-` for parameter expansion, to make it consistent throughout the script?
```suggestion
    if [[ -n "${JOINER_SST_DIR:-}" && -z "$WSREP_LOG_DEBUG" ]]; then
```